tern-ontology/conceptual-information-model.md

```diff
@@ -217,22 +212,22 @@ It is later sent to the local state/territory herbarium for identification.
 
 ## Site
 
-Sites are the main _ultimate_ features of interest in the context of plot-based surveys and their activities. 
+Sites are usually the main _ultimate_ features of interest in the context of plot-based surveys and their activities. 
 Acts of observations and sampling made in a site are generally undertaken in order to represent the overall state of the site. 
 
 Sites may contain nested 'sites' using the sub-sample relationship. 
 The site-type is indicated using a value from a controlled vocabulary. 
-Common site-types may include `plot`, `transect`, `quadrat`. 
-Other features identified within a Site are also represented using sub-sample relationships.
+Common site-types may include `plot`, `transect`, `quadrat`. TERN manages a list of [site types](http://linked.data.gov.au/def/tern-cv/74aa68d3-28fd-468d-8ff5-7e791d9f7159) as a SKOS controlled vocabulary.
+Other features and samples defined within a site are connected back to the site through the `is sample of` relationship.
 
 A Site may itself be a kind of Sample, as it may be deemed to be representative of a
-- fiat area such as a parcel, a local government area, state or territory
+- flat area such as a parcel, a local government area, state or territory
```

Comment:
'fiat' is correct. It means that the thing is defined by drawing lines on a map, in contrast to by some natural phenomena.

Reply:
Haha oops, sorry, I'll fix that!